content/22.vaccines-trad.md

```diff
@@ -458,6 +479,34 @@ In general, deciding on the prioritization and allocation of the COVID-19 vaccin
 Various frameworks, models, and methods have been proposed to tackle these issues with many countries, regions, or U.S. states devising their own distribution and administration plans [@doi:10.1126/science.abe6959; @doi:10.1038/s41598-020-78447-3; @doi:10.1016/j.vaccine.2020.12.059; @doi:10.1136/medethics-2020-107036; @doi:10.1101/2021.01.12.21249694].
 The majority of the distribution plans prioritized offering vaccines to key workers such as health care workers and those who are clinically vulnerable, such as the elderly, the immunocompromised, and individuals with comorbidities, before targeting the rest of the population, who are less likely to experience severe outcomes from COVID-19 [@url:https://ourworldindata.org/covid-vaccination-policy].
 The availability of vaccines developed in a variety of countries using a variety of platforms is likely to work in favor of worldwide vaccine access.
+The initiative by Texas Children’s Hospital and Baylor College of Medicine to develop Corbevax, a patent-free COVID-19 protein subunit vaccine, is an important step towards vaccine equity because the manufacturing specifications can be shared globally.
```

Comment:
I'm not sure why your vaccine hesitancy text is showing up green, as that PR merged today! I don't think I changed it at all.

Reply:
Ummm I don't think I did anything but who knows...

Reply:
Oh 100% not your fault @RLordan! When I work locally on my computer in git and then sync with GitHub, sometimes it seems like it takes a minute for it to catch up when the branch updates? But I don't know why that would happen.